TKeazirian.HTTPServer/Router/Routes.cs

```diff
@@ -0,0 +1,34 @@
+namespace TKeazirian.HTTPServer.Router;
+
+using Response;
+using Request;
+
+public class Routes
+{
+    private readonly Dictionary<string, Route> _routes;
+
+    public Routes()
+    {
+        _routes = new Dictionary<string, Route>();
+    }
```

Comment:
Is `Routes` just a `List<Route>` in disguise?  Could you accomplish everything here by just using a List?

Reply:
That is, assuming a `Route` includes a `path`, which it appears it no longer does.

Reply:
IF you added it back, could you just use a `List<Route>` and drop this class though?

Reply:
that was my original question; not why the `path` was removed.

Reply:
Hmm...tbh I'm not sure but I will look at it to see!

Reply:
So I played around with this - got it working, unit tests passing but acceptance tests not passing.

I may be approaching it in an unexpected way, but it feels more complicated to do it this way?  I think I would have to loop through the whole List<Route> and then check each one according to the logic , as opposed to the way now which feels more direct.  Especially if I remove the path from the Route object.